### PR TITLE
Add multi-step form for certificate requests

### PR DESCRIPTION
**Pull request by aar0npal**
Major changes:
- Added multi-step form in `pages/services` page
- Locally saves photos to `<resident_id>/` 

Note: New packages are installed and some packages are updated

### CODE REVIEW

**Comment by webzxx** [comment]
Approved

File: prisma/schema.prisma [hunk 1]
```diff
@@ -119,10 +119,10 @@ model Address {
   id              Int           @id @default(autoincrement())
   residentId      Int           @unique
   residencyType   ResidencyType
-  yearsInMolinoIV Int
+  yearsInMolinoIV Int?
```
Comment (webzxx) [inline prisma/schema.prisma:123]: yearsInBahayToro

File: prisma/schema.prisma [hunk 2]
```diff
@@ -119,10 +119,10 @@ model Address {
...
   blockLot        String?
   phase           String?
-  street          String
+  street          String?
   subdivision     String
   barangay        String        @default("Molino IV")
```
Comment (webzxx) [inline prisma/schema.prisma]: `barangay String @default("Bahay Toro")`

File: prisma/schema.prisma [hunk 3]
```diff
@@ -119,10 +119,10 @@ model Address {
...
   city            String        @default("Bacoor")
```
Comment (webzxx) [inline prisma/schema.prisma]: `city String @default("Quezon City")`

File: prisma/schema.prisma (hunk 1 above)
Comment (webzxx) [inline prisma/schema.prisma]: `yearsInBahayToro Int?`

File: prisma/schema.prisma [hunk 4]
```diff
@@ -85,6 +85,7 @@ model User {
 
 model Resident {
   id             Int         @id @default(autoincrement())
+  molinoSystemId String      @unique @default(dbgenerated("concat('MOLINO-IV-', to_char(nextval('molino_system_id_seq'), 'FM00000'))"))
```
Comment (webzxx) [inline prisma/schema.prisma]: `bahayToroSystemId String @unique @default(dbgenerated("concat('BAHAY-TORO-', to_char(nextval('bahayToro_system_id_seq'), 'FM00000'))"))`

File: prisma/schema.prisma [hunk 5]
```diff
@@ -164,3 +166,12 @@ model Certificate {
   createdAt  DateTime @default(now())
   updatedAt  DateTime @updatedAt
 }
+
+// These are used to generate sequential numbers for molinoSystemId and referenceNumber
+model MolinoSystemIdSequence {
```
Comment (webzxx) [inline prisma/schema.prisma]: `model BahayToroSystemIdSequence {`

File: src/components/form/CertificateForm.tsx [hunk 6]
```diff
@@ -23,48 +25,50 @@ const steps = [
...
+    status: undefined,
+    sector: undefined,
+    emergencyContactName: "",
+    emergencyRelationship: "",
+    emergencyContact: "",
+    emergencyContactAddress: "",
+  },
+  address: {
+    residency: undefined,
+    yearsInMolinoIV: 0,
```
Comment (webzxx) [inline src/components/form/CertificateForm.tsx]: `yearsInBahayToro: 0,`

**Comment by aar0npal** [comment]
All completed @webzxx 
Will make a new PR for admin dashboard

**Pull request merged**